### PR TITLE
added Raspberry Pi override for /dev/gpiomem device (fix #8)

### LINKED ISSUE #8: custom device "/dev/gpiomem": no such file or directory
It seems like `docker-compose.yml` is built more towards Raspberry Pis, but the `/dev/gpiomem` kernel device does not exist in `amd64` Linux computers.

Since there is the ability to specify the Docker-Compose filename, can we create a file like `rpi-compose.yml` and remove the `/dev/gpiomem` device in the original `docker-compose.yml`? Removing (or commenting) [docker-compose.yml#L108](https://github.com/smartgic/docker-mycroft/blob/f05cb579ee224c00a1fe7044908ec21021d17053/docker-compose.yml#L108) seems to make it work just fine on Docker under Ubuntu 20.04 LTS.

Thanks in advance!

### PR DESCRIPTION
**Pull request by bryanpedini**
Like title says.

Currently tested working on Kubuntu 20.04 representing `x86_64` (`amd64`) and a Raspberry Pi 3b+.
I could need someone with a Raspberry Pi 4 to provide me the result of `uname -m` from console.

### CODE REVIEW

**Comment by goldyfruit** [comment]
> Like title says.
> 
> Currently tested working on Kubuntu 20.04 representing `x86_64` (`amd64`) and a Raspberry Pi 3b+.
> I could need someone with a Raspberry Pi 4 to provide me the result of `uname -m` from console.

```
pi@smartgic-pi4b8-13:~ $ uname -m
aarch64
```

**Comment by goldyfruit** [comment]
You will have to `rebase` your commit due to the merged of #9.

**Comment by bryanpedini** [comment]
> You will have to `rebase` your commit due to the merged of #9.

I think they can be squashed together, I didn't touch in one branch what I touched on the other and vice versa.

Could you please test the last commit on a rPi4 to confirm it working?

File: run.sh [hunk 1]
```diff
@@ -71,6 +71,15 @@ for COMMAND in "docker" "docker-compose"; do
     command_exists "${COMMAND}"
 done
 
+# Check if platform is Raspberry Pi, to apply it's own configuration later
+# This is done since amd64 computers do not have the same hardware device
+#  for GPIO pins like the Pi does.
+case $(uname -m) in
```
Comment (goldyfruit) [inline run.sh:77]: I got an error here:
```
./run.sh: line 79: syntax error near unexpected token `)'
./run.sh: line 79: `    aarch64)'
```
I would use:
```bash
case $(uname -m) in
    armv7l | aarch64)
        raspberrypi="true" 
        ;;
```

**Review by goldyfruit — CHANGES_REQUESTED** [review]

File: run.sh (hunk 1 above)
Comment (bryanpedini) [reply run.sh:77]: Sure, I don't actually remember how to use cases in bash, I use them so little 😅

**Review by bryanpedini — COMMENTED** [review]
Should be fixed with last commit

**Review by goldyfruit — APPROVED** [review]

**Comment by goldyfruit** [comment]
Thanks for the PR @bryanpedini :+1:

**Pull request merged**